src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -13,10 +13,14 @@
     <TargetFrameworkProfile>Profile7</TargetFrameworkProfile>
     <FileAlignment>512</FileAlignment>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <SccProjectName>SAK</SccProjectName>
-    <SccLocalPath>SAK</SccLocalPath>
-    <SccAuxPath>SAK</SccAuxPath>
-    <SccProvider>SAK</SccProvider>
+    <SccProjectName>
```

Comment:
:exclamation: These properties should be removed (by editing the project file) rather than just being set to empty.